checkov/common/graph/graph_builder/graph_components/blocks.py

```diff
@@ -144,7 +144,7 @@ def update_inner_attribute(
     def get_export_data(self) -> Dict[str, Union[bool, str]]:
         return {"type": self.block_type, "name": self.name, "path": self.path}
 
-    def get_base_attributes(self) -> Dict[CustomAttributes, Union[str, List[str], Dict[str, Any]]]:
+    def get_base_attributes(self) -> Dict[str, Union[str, List[str], Dict[str, Any]]]:
```

Comment:
@rotemavni my fault, I thought `CustomAttributes` is an Enum, you were right with `str` type 💪